source/reference/minio-mc/mc-undo.rst

```diff
@@ -59,29 +60,56 @@ Parameters
    The full path to the object or prefix where the command should run.
    The path must include the :ref:`ALIAS <minio-mc-alias>`, bucket, and prefix or object name.
 
-.. mc-cmd:: --last
+.. mc-cmd:: --action
    :optional:
 
-   Accepts an integer value specifying the number of ``PUT`` and/or ``DELETE`` changes to undo.
-   
-   If not specified, the command undoes one (``1``) operation.
+   Undo the most recent change of the specified type.
+   Accepted values are ``DELETE`` or ``PUT``.
 
-.. mc-cmd:: --recursive, r
+   By default, :mc:`mc undo` reverses both ``DELETE`` and ``PUT`` operations.
+   Use :mc-cmd:`~mc undo --action` to choose one or the other, but only for the most recent operation of the specified type.
+
+   The following command undoes the most recent ``PUT`` for the file ``today.zip`` in bucket ``data``:
```

Comment:
Maybe? `undoes` feels ... awkward?

```suggestion
   The following command reverses the most recent ``PUT`` operation for the file ``today.zip`` in bucket ``data``:
```

Reply:
The existing text uses `undoes` but yeah I was wondering about that. I think I should revisit throughout.